www/main/menu.js

```diff
@@ -172,10 +172,9 @@ const hukamnamaButton = h(
     'a.hukamnama-button',
     {
       onclick: () => {
-        getJSON('https://api.banidb.com/hukamnama/today', (error, response) => {
+        getJSON('https://api.banidb.com/v2/hukamnamas/today ', (error, response) => {
```

Comment:
/today does not add anything. the api is just giving you the latest. I'd leave it out and just call ```https://api.banidb.com/v2/hukamnamas```

Reply:
I don't agree with that. its clear from readability yes, but it gives false pretense on what the API provides. until the API changes we shouldn't be calling it in a way that it's not intended, even if it's the result we want. 

I'd prefer if we leave out "today" since the actual API uses what after the path and expect it to be a date.

Reply:
@ManjotS do you know when we will have https://api.banidb.com/v2/hukamnamas/{date} functionality?

Reply:
from @ManjotS "We have had that for years and its what drives the yesterday/tomorrow
buttons on the hukamnama sttm web

https://api.banidb.com/v2/hukamnamas/2020/07/15"

Reply:
@ManjotS /yesterday and /today give the same result though... a bug?